### PR TITLE
many: assorted typos

### PR DESCRIPTION
**Pull request by zyga**
Typos I've ran across while looking at the code last week.

### CODE REVIEW

**Review by MiguelPires — APPROVED** [review]

File: overlord/snapstate/policy/export_test.go [hunk 1]
```diff
@@ -16,7 +16,7 @@ var (
 	ErrSnapdNotRemovableOnCore       = errSnapdNotRemovableOnCore
 	ErrSnapdNotYetRemovableOnClassic = errSnapdNotYetRemovableOnClassic
 
-	ErrEphemeralSnapsNotRemovalable = errEphemeralSnapsNotRemovalable
+	ErrEphemeralSnapsNotRemovalable = errEphemeralSnapsNotRemovable
```
Comment (ZeyadYasser) [inline overlord/snapstate/policy/export_test.go:19]:
```suggestion
	ErrEphemeralSnapsNotRemovable = errEphemeralSnapsNotRemovable
```

**Review by ZeyadYasser — APPROVED** [review]
LGTM, Thanks! small nitpick

**Review by andrewphelpsj — APPROVED** [review]
Looks good, once you get in @ZeyadYasser's suggestion.

File: overlord/snapstate/policy/export_test.go (hunk 1 above)
Comment (zyga) [reply overlord/snapstate/policy/export_test.go:19]: Thanks, I force-pushed this as the rename was broader due to symbol usage.

**Pull request merged**